plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/preferences/GoogleLandingPage.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2015 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.eclipse.dataflow.ui.preferences;
+
+import org.eclipse.core.runtime.IAdaptable;
+import org.eclipse.jface.preference.PreferencePage;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.ui.IWorkbench;
+import org.eclipse.ui.IWorkbenchPreferencePage;
+import org.eclipse.ui.IWorkbenchPropertyPage;
+
+/**
+ * An empty PreferencePage to construct a Google category.
+ */
+public class GoogleLandingPage
+    extends PreferencePage implements IWorkbenchPreferencePage, IWorkbenchPropertyPage {
+  private IAdaptable element;
+
+  @Override
+  public IAdaptable getElement() {
+    return element;
+  }
+
+  @Override
+  public void setElement(IAdaptable element) {
```

Comment:
Just spitballing, but it bugs me a little about Eclipse that IAdaptables aren't more type safe. Seems like we ought to throw the exception here if it's the wrong type rather than later when we try to convert the object.